backbone/src/main/java/org/researchstack/backbone/answerformat/TextAnswerFormat.java

```diff
@@ -1,16 +1,21 @@
 package org.researchstack.backbone.answerformat;
 
 
+import android.text.InputType;
+
 /**
  * The TextAnswerFormat class represents the answer format for questions that collect a text
  * response from the user.
  */
 public class TextAnswerFormat extends AnswerFormat
 {
     public static final int UNLIMITED_LENGTH = 0;
-    private int maximumLength;
+    int maximumLength;
```

Comment:
My default is to keep the language default protected, I guess it's just habit.  But I understand the benefit in a library to keep member variables private as well.  I'll switch these to private.